### PR TITLE
Document and slightly refactor sir() and gen_sir()

### PR DESCRIPTION
**Pull request by grahamalama**
- Make n parameter optional, because it can be calculated by summing s + i + r
- Make variable names a little more self explanatory

### CODE REVIEW

**Comment by grahamalama** [comment]
I probably got some argument / parameter names wrong, so please tell me how I should change them if needed! I did my best by Googling / searching for context in elsewhere in the project.

File: src/penn_chime/models.py [hunk 1]
```diff
@@ -235,44 +235,72 @@ def get_growth_rate(doubling_time: Optional[float]) -> float:
...
+        susceptible: Number of susceptible individuals
+        infected: Number of infected individuals
+        recovered: Number of recovered individuals
+        gamma: The rate an infected recovers and moves into the resistant phase
+        i_day: The number of days after the first hospitalization
+        policies: A sequences of tuples that represent a new value of beta (how
+            often a susceptible-infected contact results in a new infection),
+            and how many days that value of beta will last.
+    
+    Returns:
```
Comment (jlubken) [inline src/penn_chime/models.py:304]: Yields:
Tuples representing the day and the projected number of..., where day 0 is the current_date of the simulation.
Returns:
A Generator of such tuples.

Comment (grahamalama) [reply src/penn_chime/models.py:304]: Updated! Nice catch

**Comment by jlubken** [comment]
@grahamalama Phil has optimized this code quite a bit for this to be merged.

This penalizes every iteration of what is essentially an inner loop function to add a check for n: `n: Optional[int] = None`.

**Comment by grahamalama** [comment]
Okay -- I'll close this PR and maybe reopen another. I'll leave the "refactoring" out and just focus on docstrings / variable names

**Pull request closed without merging**